xapian-core/include/xapian/cluster.h

```diff
@@ -605,6 +611,14 @@ class XAPIAN_VISIBILITY_DEFAULT KMeans : public Clusterer {
      */
     ClusterSet cluster(const MSet &mset);
 
+    /** Set the Xapian::Stopper object to be used for identifying stopwords.
+     *  Stopwords are discarded while calculating term frequency for terms
+     *
```

Comment:
There should really be a blank (except for the `*`) line between the brief description and the rest of the doc comment.

I missed this before, so rather than delay the merge, I've merged and then fixed it as a follow-up commit: 659b03cde562b55fb080dc9bb8c6114683e3a752